### DISCUSSION TITLE
Exporting mp3 files with an existing preset

### DISCUSSION THREAD

**Discussion by CoreyTV** [wwmm/easyeffects#1465] [Ideas]
This would be an awesome feature if it wouldn't be terribly hard to do. If I have a (Car-Audio) profile, I could export an entire folder of mp3's to a new folder of mp3's with my Car-Audio profile, then I could save it to a USB to plug into my car and enjoy the effects outside of my linux machine! I absolutely love the bass enhancer and crystalizer.

**Comment by wwmm** [comment]
>if it wouldn't be terribly hard to do

I think someone opened an issue in the past requesting application of effects to files. The problem is that we always had live processing as target. So it won't be straightforward to add offline file processing to EasyEffects. I am not sure if it is even possible as things are right now.

**Reply by CoreyTV** [reply]
I see. Gotcha. I may end up configuring a raspberry pi to be a Bluetooth adapter for my car and have it live process through that instead. Thanks for the fast response!

**Comment by vchernin** [comment]
As a workaround you could manually play the audio, input that audio being played to EasyEffects, and save EE's output to an MP3.

That could be scripted, but if you have an hour of audio files it would take an hour to process it, not exactly ideal.
